src/python/qepsi4/_psi4_test.py

```diff
@@ -77,28 +187,53 @@ def test_run_psi4_using_n_occupied_extract():
     assert hamiltonian.n_qubits == 8
 
     qubit_operator = qubit_operator_sparse(jordan_wigner(hamiltonian))
-    energy, state = jw_get_ground_state_at_particle_number(qubit_operator, 4)
+    energy, state = jw_get_ground_state_at_particle_number(qubit_operator, 4) 
     assert math.isclose(energy, -14.654620243980217)
 
-def test_run_psi4_using_n_occupied_extract_with_rdms():
+def test_run_psi4_using_n_occupied_extract_with_rdms_0():
     results, hamiltonian, rdm = run_psi4(
         dilithium_geometry,
         method="fci",
         basis="STO-3G",
         n_active_extract=4,
-        n_occupied_extract=2,
+        n_occupied_extract=1,
```

Comment:
Any reason for this switch? It could be nice to have 2 occupied orbitals.

Reply:
I think the previous version of the test was actually incorrect because I confused `n_occupied_extract` with the number of doubly occupied (core) orbitals.